src/nxdoc/web-ui/web-ui-tutorials/web-ui-slots.md

```diff
@@ -289,7 +299,7 @@ This slot is displayed when selecting one or more children documents of a folder
 
 {{#> callout type='note' }}
 - Until Web-UI version `0.9`, this slot was called `DOCUMENT_LISTING_ACTIONS` and its properties may have changed.
-{{/callout}}
+  {{/callout}}
```

Comment:
Looks like the spaces are not necessary here and only add noise to the commit, could be removed for a cleaner PR

Reply:
Done